web/styles/app_components.css

```diff
@@ -316,6 +316,10 @@ input::placeholder {
     }
 }
 
+#admin-user-list .tab-switcher .ind-tab {
+    width: 110px;
+}
```

Comment:
This seems like it belongs in `settings.css`, since it's styling for a specific element, not a reusable component.

Reply:
Fixed!

Reply:
This can be removed now, right?